cmake/Platform/Linux/CompilerSettings_linux.cmake

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
nit: Should this have # {BEGIN_LICENSE} etc

Reply:
it should not, and the one above should not have it neither, is just from the template, will remove the one from above